src/app/combo/combo.sample.html

```diff
@@ -101,6 +102,7 @@ <h4>Display Density</h4>
             <h4>Search Input</h4>
             <igx-switch [(ngModel)]="filterableFlag">Filterable</igx-switch><br/>
             <igx-switch [(ngModel)]="customValuesFlag">Custom Values</igx-switch>
+            <igx-switch [(ngModel)]="autoFocusSearch">Suppress Focus</igx-switch>
```

Comment:
Maybe revert logic here or rename "Suppress Focus" to "Allow Search Focus" as currently the switch does the opposite.